docs/GettingStarted.asciidoc

```diff
@@ -340,9 +340,9 @@ The personal configuration should be stored in a file
 the `client.conf`, i.e. sections for each machine, e.g. `localhost`.
 
 [[get-testing]]
-== Testing openSUSE or Fedora
+== Testing openQA on openSUSE or Fedora
```

Comment:
This completely reverses meaning of title, 

This isn't about testing openQA on some distribution but about testing distro in openQA

Reply:
Oops! Sorry my fault. 😔

Reply:
If you find it helpful you could phrase it the other way around:

```suggestion
== Testing openSUSE or Fedora with openQA
```